front/src/routes/scene/edit-scene/actions/DeviceSetValue.jsx

```diff
@@ -136,6 +192,22 @@ class DeviceSetValue extends Component {
               <Text id="global.requiredField" />
             </span>
           </label>
+          <div className="nav-tabs" style={{ display: 'flex', margin: 0 }}>
+            <span
+              class={`nav-link ${this.state.computed ? '' : 'active'}`}
+              style={'width: 50%'}
```

Comment:
For the style, as you use lots of styles, could you create a .css file and regroup all styles in a file?

I know that sometimes in the frontend we use inline-styles, but it's a bad practice and I think we should move away from it